pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -361,7 +362,35 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 			ObjectMeta: kapi.ObjectMeta{
 				Name: JobControllerRoleName,
 			},
-			Rules: []authorizationapi.PolicyRule{},
+			Rules: []authorizationapi.PolicyRule{
+				// JobController.jobController.ListWatch
+				{
+					APIGroups: []string{authorizationapi.APIGroupExtensions},
+					Verbs:     sets.NewString("list", "watch"),
+					Resources: sets.NewString("jobs"),
+				},
+				// JobController.syncJob() -> updateJobStatus()
+				{
+					APIGroups: []string{authorizationapi.APIGroupExtensions},
+					Verbs:     sets.NewString("update"),
+					Resources: sets.NewString("jobs/status"),
+				},
+				// JobController.podController.ListWatch
+				{
+					Verbs:     sets.NewString("list", "watch"),
+					Resources: sets.NewString("pods"),
+				},
+				// JobController.podControl (RealPodControl)
+				{
+					Verbs:     sets.NewString("create", "delete"),
```

Comment:
> sort of surprised we don't need get pods... does this mean the job controller always lists pods to see if the pod it wants to create already exists (which seems inefficient) or speculatively does a create and tolerates an AlreadyExists error?

list+watch allows the controller to maintain a cache.  I assumed that's what it did.